### ISSUE TITLE
feat: Passing the content of the ai version of the editor

### ISSUE THREAD

**Issue by alirezashalchian** [steven-tey/novel#471]
### Describe the feature you'd like to request

how can i pass the editor content (everything a user types in the editor, or images, or embedded videos, with the exact same styles)

### Describe the solution you'd like to see

assume i'm gonna use the editor for a blogging website, how can i pass whatever that is written in the editor to a single blog post when users post what they have written in the editor.
i would be grateful if anyone helped me with this.
thanks in advance.

**Comment by Trimud** [comment]
Hi, you need to set the `editable` attribute to `false` in `EditorContent` component.

**Comment by alirezashalchian** [comment]
Hi @Trimud i really appreciate you taking time to answer,
so basically what i do is to use non-editable `EditorContent` inside any div i want ?

**Comment by Trimud** [comment]
Yes, that's the point. But keep in mind that this would be rendered client side as the editor uses "use client"

Another option is to follow this https://github.com/steven-tey/novel/issues/456#issuecomment-2439921423

I am currently looking for a way to have this implemented in my prject but it makes me wonder why do I not use TipTap directly without Novel https://github.com/ueberdosis/tiptap/pull/5528

**Comment by AKRking** [comment]
@Trimud  hi did you manage to do static rendering on novel?what did you do to overcome this issue, pls elaborate
